Contributing.md

```diff
@@ -0,0 +1,65 @@
+This website is maintained by the in-toto steering committee and is modeled after the [Kubernetes documentation contribution workflow](https://kubernetes.io/docs/contribute/).
```

Comment:
Is this contributing doc ever going to be on a website?
maybe appearing somewhere here? https://in-toto.io/specs/

Or will it always be a contributing guide for this repo?

Reply:
The plan is for the content of the Spec to be transformed into a Docs section of the in-toto website. I expect this repo will continue to be the source for the text, even if that text is broken up into separate files to match separate web pages. 
We expect to add navigation (site and page TOCs) to make it easier for new users to find introductory information before diving into the demo.

Reply:
I think it also depends on the final structure of in-toto repositories. Are we going to rename this one to in-toto/specification and then create a new in-toto/docs repository? Is the specification going to continue to live in the docs repository?